include/lsst/meas/algorithms/CoaddPsf.h

```diff
@@ -71,6 +71,8 @@ class CoaddPsf : public afw::table::io::PersistableFacade<CoaddPsf>, public Imag
     /// Polymorphic deep copy.  Usually unnecessary, as Psfs are immutable.
     virtual PTR(afw::detection::Psf) clone() const;
 
+    /// Return a clone with specified kernel dimensions
+    virtual PTR(afw::detection::Psf) resized(int width, int height) const;
```

Comment:
Should be `override` instead of `virtual`, here and below (but see comment on `afw` PR about separate commits).